src/Filesystem_TEST.cc

```diff
@@ -293,27 +200,29 @@ TEST_F(FilesystemTest, fileOps)
 /// \brief Test file operations
 TEST_F(FilesystemTest, moreFileOps)
 {
-  EXPECT_FALSE(common::copyFile("__wrong__.tmp", "test2.tmp"));
-  EXPECT_TRUE(!common::exists("test2.tmp"));
-  EXPECT_FALSE(common::copyFile("test.tmp", "__wrong_dir__/__wrong__.tmp"));
-  EXPECT_TRUE(!common::exists("__wrong_dir__"));
-
-  EXPECT_FALSE(common::moveFile("__wrong__.tmp", "test3.tmp"));
-  EXPECT_TRUE(!common::exists("test3.tmp"));
-  EXPECT_FALSE(common::moveFile("test2.tmp", "__wrong_dir__/__wrong__.tmp"));
-  EXPECT_TRUE(!common::exists("__wrong_dir__"));
+  EXPECT_FALSE(common::copyFile("__wrong__.tmp", "test2.tmp",
+        common::FSWO_SUPPRESS_WARNINGS));
+  EXPECT_FALSE(common::exists("test2.tmp"));
+  EXPECT_FALSE(common::copyFile("test.tmp", "__wrong_dir__/__wrong__.tmp",
+        common::FSWO_SUPPRESS_WARNINGS));
+  EXPECT_FALSE(common::exists("__wrong_dir__"));
+
+  EXPECT_FALSE(common::moveFile("__wrong__.tmp", "test3.tmp",
+        common::FSWO_SUPPRESS_WARNINGS));
+  EXPECT_FALSE(common::exists("test3.tmp"));
+  EXPECT_FALSE(common::moveFile("test2.tmp", "__wrong_dir__/__wrong__.tmp",
+        common::FSWO_SUPPRESS_WARNINGS));
+  EXPECT_FALSE(common::exists("__wrong_dir__"));
 }
 
 /////////////////////////////////////////////////
 TEST_F(FilesystemTest, exists)
 {
-  std::string new_temp_dir;
-  ASSERT_TRUE(create_and_switch_to_temp_dir(new_temp_dir));
   ASSERT_TRUE(create_new_empty_file("newfile"));
   ASSERT_TRUE(createDirectory("fstestexists"));
 
   const auto &relativeSubdir = joinPaths("fstestexists", "newfile2");
-  const auto &absoluteSubdir = joinPaths(new_temp_dir, relativeSubdir);
+  const auto &absoluteSubdir = joinPaths(cwd(), relativeSubdir);
```

Comment:
How would this play when the user has a non-standard build path? I believe it should be fine, but just something to think about.

Reply:
The test is still executing in a temp directory, but that functionality has been moved to the fixture: https://github.com/ignitionrobotics/ign-common/pull/282/files#diff-cbe626f392b0654d4e901fc284e80468c4194ab344d0187fb391203e21d65a58R43-R44

The behavior shouldn't change at all.